### DISCUSSION TITLE
Can I ask why the accuracy of the Pro version can reach 99.5%?

### DISCUSSION THREAD

**Discussion by fyjiayou** [fingerprintjs/fingerprintjs#998] [General]
Can I ask why the accuracy of the Pro version can reach 99.5%? If our company likes to use the Pro version and the number of requests per month is around 1 million, how much will it cost approximately per month?

**Comment by makma** [comment]
Hello @fyjiayou!

> Can I ask why the accuracy of the Pro version can reach 99.5%

Explanation: https://dev.fingerprint.com/docs/understanding-our-995-accuracy

>  how much will it cost approximately per month

Pricing info: https://fingerprint.com/pricing/
